community/server/src/test/java/org/neo4j/server/rest/CreateRelationshipDocTest.java

```diff
@@ -34,18 +34,13 @@
 import org.neo4j.test.GraphDescription.Graph;
 import org.neo4j.test.TestData.Title;
 
-import com.sun.jersey.api.client.ClientResponse.Status;
 import static org.junit.Assert.assertTrue;
 
 public class CreateRelationshipDocTest extends AbstractRestFunctionalDocTestBase
 {
-    /**
-     * Upon successful creation of a relationship, the new relationship is
-     * returned.
-     */
     @Test
     @Graph( "Joe knows Sara" )
-    @Documented
+    @Documented( "Upon successful creation of a relationship, the new relationship is returned." )
     @Title( "Create a relationship with properties" )
```

Comment:
I think we should do as little as possible around the REST API docs and only change things if there's a need for it!